content/modules/install/add-module-to-cluster.md

```diff
@@ -45,7 +45,8 @@ To add a module to the cluster using the REST API:
 1. Add the module to the cluster with a [`POST` request to the `/v1/modules`]({{<relref "/rs/references/rest-api/requests/modules#post-module">}}) endpoint:
 
     ```sh
-    curl -k -u "admin@redislabs.com:<password>" -F "module=@/tmp/redisearch.Linux-ubuntu16.04-x86_64.2.2.6.zip" https://localhost:9443/v1/modules
+    POST https://[host][:port]/v1/modules
+    "module=@/tmp/redisearch.Linux-ubuntu16.04-x86_64.2.2.6.zip"
```

Comment:
When you remove the -F cURL option, I think it becomes unclear that "module=@/tmp/redisearch.Linux-ubuntu16.04-x86_64.2.2.6.zip" needs to be specified as form-data rather than another type of request body.